test/e2e/reconfigure_test.go

```diff
@@ -49,6 +49,10 @@ func reconfigurationTest(t *testing.T) {
 	_, err = testCtx.client.K8s.CoreV1().Nodes().Patch(context.TODO(), machineNodes[0].Name, types.JSONPatchType,
 		patchData, metav1.PatchOptions{})
 	require.NoError(t, err)
+	// Ensure operator communicates to OLM that upgrade is not safe when processing Machine nodes
+	err = testCtx.validateUpgradeableCondition(metav1.ConditionFalse)
```

Comment:
These test seem timing dependent. Are you sure they are not flaky?

Reply:
I have not seen any failures locally or in the e2e tests. Waiting for the condition to be reflected in status should prevent any timing issues I thought.